proto/p4/p4runtime.proto

```diff
@@ -399,11 +400,27 @@ message RegisterEntry {
   P4Data data = 3;
 }
 
+// Used to configure the digest extern only, not to stream digests or acks
+message DigestEntry {
+  uint32 digest_id = 1;
+  // a DigestList message is streamed when the following conditions are met:
+  //   - there is at least one digest ready
+  //   - the oldest digest in the list has been waiting for at least timeout_ns
```

Comment:
It isn't clear whether timeout_ns on previous line is meant to refer to max_timeout_ns or ack_timeout_ns.  Maybe max_timeout_ns is intended?

Reply:
Clarified this by using `max_timeout_ns`.